clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/ConsumptionTask.java

```diff
@@ -71,7 +71,8 @@ public ConsumptionTask(
     this.recordsThrottler = recordsThrottler;
     this.stats = stats;
     this.cleaner = cleaner;
-    this.logger = LogManager.getLogger(getClass().getSimpleName() + "[ " + kafkaUrl + " - " + taskId + " ]");
+    String kafkaUrlForLogger = kafkaUrl.replace(".", "_");
```

Comment:
NIT: Shall we put it into an Util static method?

Reply:
Yeah. Hopefully it'll help with the code coverage 🙈